src/markdown-script.ts

```diff
@@ -1,14 +1,46 @@
-/**
- * This script runs in VSCode markdown builtin preview webview
- */
+
 import { pintoraStandalone } from '@pintora/standalone'
 
 let hasThemeBeSet = false
+const openedErrors = {}
+
+const addCSS = css => (document.head.appendChild(document.createElement('style')).innerHTML = css)
+addCSS(`
+  .pintora-error pre {
+   color: var(--vscode-errorForeground);
+  }
+  .pintora-error pre {
+    display: none;
+  }
+  .pintora-error.open pre {
+   display: block;
+   color: #ff4c4c;
```

Comment:
This should be `color: var(--vscode-errorForeground);` as well, right ?

Reply:
Hello, I already updated it